go/vt/discovery/healthcheck.go

```diff
@@ -517,7 +692,8 @@ func (hc *HealthCheckImpl) checkConn(hcc *healthCheckConn, name string) {
 	defer hc.finalizeConn(hcc)
 
 	// Initial notification for downstream about the tablet existence.
-	hc.updateHealth(hcc.tabletStats.Copy(), hcc.conn)
+	// do not copy
+	hc.updateHealth(&hcc.tabletStats, hcc.conn)
```

Comment:
I don't think `hc` should be updating health. Will be better if stream owned the tabletstats, and kept that up-to-date. `hc` should just consume it.

Reply:
deleted this function